nengo/builder/learning_rules.py

```diff
@@ -86,10 +97,30 @@ def make_step(self, signals, dt, rng):
         theta = signals[self.theta]
         delta = signals[self.delta]
         alpha = self.learning_rate * dt
+        period = (1 if self.apply_every is None else
```

Comment:
Yeah, this is one of those cases where I should have made a commit. *bows apologetically*

I'll try and make one tonight.